src/linker/configuration.py

```diff
@@ -85,3 +85,12 @@ def _load_input_data_paths(input_data: Path) -> List[Path]:
         if missing:
             raise RuntimeError(f"Cannot find input data: {missing}")
         return file_list
+
+    @staticmethod
+    def _get_spark_requests(
+        environment: Dict[str, Union[Dict, str]]
+    ) -> Optional[Dict[str, Any]]:
+        spark = environment.get("spark", None)
+        if not "keep_alive" in spark:
```

Comment:
This seems like a strange mechanism to have a default for just one key. Can we just give all of these defaults?

Reply:
Not a bad idea - I'll make a ticket